docs/references/api/aggregate_functions.rst

```diff
@@ -286,3 +288,58 @@ The result will be the same as if the aggregations were applied to columns from
       "min": "2016-02-11"
     }
   ]
+
+Using Aggregates in a To-Many Spread Resource
+---------------------------------------------
+
+Unlike the to-one spreads, the columns inside a :ref:`to-many spread relationship <spread_to_many_embed>` are not treated as if they were part of the top-level resource.
+The aggregates will be done :ref:`in the context of the to-many spread resource <aggregate_functions_embed_context>`.
+For example:
```

Comment:
These special cases make the feature really confusing. And this problem is a consequence of supporting multiple columns (https://github.com/PostgREST/postgrest/pull/3640#issuecomment-2529571928), if we supported only a single column the feature would be simple.

And if users wanted multiple columns, they could still do:

```
...orders(amount),...orders(order_date)
```

Which already "conveys" why an order is necessary.

Reply:
Maybe some of the work it has been done here can be reused as a form of query optimization though. To not repeat work when two `...orders()` are done.

But I would strongly suggest trying to eliminate these special cases.

Reply:
> I think this is a documentation issue. From looking at the test-cases, this felt very natural now.

Yes I think the documentation needs to be improved. Maybe we could start with the "only aggregates" example, as it works exactly the same as to-one spreads. After that, we'd mention this case, and reword it to something like: "If you need to group by the columns inside the spread embed, then...`, then give the example.

> ...orders(amount.sum()),...orders(order_date)

(I added `.sum()` to the example) This can still be an alternative to not group by the columns. Just noticed that this doesn't work right now, because we don't allow aliases to the spread embed names. I'll fix that in another commit.

Reply:
Docs are looking much better now. I think one problem tangential to this PR is that https://docs.postgrest.org/en/v12/references/api/aggregate_functions.html are too verbose. I made a pass at reducing verbosity on https://github.com/PostgREST/postgrest/commit/33b69b5894f1e60de2e4187c6f927d6bdbc45aa3.

Reply:
> I think one problem tangential to this PR is that https://docs.postgrest.org/en/v12/references/api/aggregate_functions.html are too verbose.

I gave it a go here: https://github.com/PostgREST/postgrest/pull/3913/files#diff-3de50a0e1c505ab011e94967475d27743c244ea40c024e6bc8992fc812086285R172-R173